src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -114,7 +114,7 @@ public function register(Container $app)
         };
 
         $app['security.encoder.digest'] = function ($app) {
```

Comment:
bonus, $app['security.encoder.bcrypt.cost'] could be used to configure/override the default encoder cost